[2023-06-01T11:14:03.275+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table scheduled__2021-12-21T00:00:00+00:00 [queued]>
[2023-06-01T11:14:03.341+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table scheduled__2021-12-21T00:00:00+00:00 [queued]>
[2023-06-01T11:14:03.343+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:14:03.346+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:14:03.347+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:14:03.436+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2021-12-21 00:00:00+00:00
[2023-06-01T11:14:03.451+0000] {standard_task_runner.py:55} INFO - Started process 105 to run task
[2023-06-01T11:14:03.493+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'create_postgres_table', 'scheduled__2021-12-21T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmpb_5rr2je']
[2023-06-01T11:14:03.498+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask create_postgres_table
[2023-06-01T11:14:03.707+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table scheduled__2021-12-21T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:14:03.998+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2021-12-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-21T00:00:00+00:00
[2023-06-01T11:14:04.000+0000] {sql.py:253} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2023-06-01T11:14:04.023+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:14:04.854+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:14:14.870+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 261, in execute
    return_last=self.return_last,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 331, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/hooks/postgres.py", line 144, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "host.docker.local" to address: Temporary failure in name resolution

[2023-06-01T11:14:14.883+0000] {taskinstance.py:1327} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_with_postgres_operator_v03, task_id=create_postgres_table, execution_date=20211221T000000, start_date=20230601T111403, end_date=20230601T111414
[2023-06-01T11:14:14.902+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 10 for task create_postgres_table (could not translate host name "host.docker.local" to address: Temporary failure in name resolution
; 105)
[2023-06-01T11:14:14.935+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-06-01T11:14:14.971+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
